modules/ossm-release-2-6-1.adoc

```diff
@@ -0,0 +1,37 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-2-6-1_{context}"]
+= {SMProductName} version 2.6.1
+
+This release of {SMProductName} includes 2.6.1, 2.5.4, and 2.4.10, addresses Common Vulnerabilities and Exposures (CVEs), contains a bug fix, and is supported on {product-title} 4.14 and later.
+
+include::snippets/ossm-current-version-support-snippet.adoc[]
+
+The most current version of the {KialiProduct} can be used with all supported versions of {SMProductName}. The version of {SMProductShortName} is specified by using the `ServiceMeshControlPlane` resource.  The version of {SMProductShortName} automatically ensures a compatible version of Kiali.
+
+[id=ossm-release-2-6-1-components_{context}]
+== Component updates
+
+|===
+|Component |Version
+
+|Istio
+|1.20.8
+
+|Envoy Proxy
+|1.28.5
+
+|Kiali
+|1.89.1
```

Comment:
Here should be also 1.73.14, the 2.6 will still use Kiali server 1.73 even though we release the newer operator

Reply:
SMCP V2.6 :     Kiali  1.89.1 Kiali ossmc 1.89.0 was in the Jira so I had update this accordingly based on the Jira.

I see that has changed now so I will make this update.